### ISSUE TITLE
Do I need a mic?

### ISSUE THREAD

**Issue by bottleblue** [speech-translator-ext/speech-translator-readme#45]
I don't have a mic. Is it required to use this extension?

**Comment by kappaflow** [comment]
No, you don't need a mic if you want to use **Setup 1. Capture your system playback audio**

**Comment by bottleblue** [comment]
I setup capture system playback audio correctly with Stereo Mix but there's no translations for Youtube videos.

**Comment by kappaflow** [comment]
Have you checked the troubleshooting guide?
https://github.com/speech-translator-ext/speech-translator-readme/blob/main/TROUBLESHOOTING.md

You can select the input audio device here as well:
chrome://settings/content/microphone

>  I don't see any microphone icon inside the address bar or anywhere else but I did grant permission for microphone when I clicked Grant Audio Permission

Can you send a screenshot?

**Comment by bottleblue** [comment]
Maybe it's because I'm using Brave browser?

**Comment by kappaflow** [comment]
Yes, the speech recognition functionality is available only in Chrome and Edge (because Google and Microsoft basically pay for it).

**Comment by bottleblue** [comment]
Oh that sucks. But thx for your help!
